### PR TITLE
Footstep sounds: 5 new sounds

### PR DESCRIPTION
**Pull request by paramat**
Original recording by Ryding.
http://freesound.org/people/Ryding/
Found by Anth0rx, edited by paramat.

Add and use 'default.node_sound_snow_defaults()' function.
//////////////

Addresses #1388 

This is a re-edit of the snow sounds found by @Anth0rx and first submitted in #1499 
I have submitted 5 sounds as before as that helps with repetition issues.
Our current sounds have only 3 versions, one sounds weird, and one is a different sample rate to the others, they are highest priority for replacing.

Sound source http://freesound.org/people/Ryding/sounds/94337/

### CODE REVIEW

**Comment by paramat** [comment]
Footstep, dig and dug sound volumes now rebalanced.

**Comment by paramat** [comment]
PR complete.

**Comment by Fixer-007** [comment]
Nice sounds.

File: mods/default/nodes.lua [hunk 1]
```diff
@@ -526,9 +526,9 @@ minetest.register_node("default:snow", {
 	},
 	groups = {crumbly = 3, falling_node = 1, puts_out_fire = 1, snowy = 1},
 	sounds = default.node_sound_dirt_defaults({
-		footstep = {name = "default_snow_footstep", gain = 0.15},
-		dug = {name = "default_snow_footstep", gain = 0.2},
-		dig = {name = "default_snow_footstep", gain = 0.2}
+		footstep = {name = "default_snow_footstep", gain = 0.2},
```
Comment (SmallJoker) [inline mods/default/nodes.lua]: This deserves a function: `default.node_sound_snow_defaults`

**Comment by SmallJoker** [comment]
These sounds are way better than the previous ones. :+1: once the comment is addressed.

File: mods/default/nodes.lua (hunk 1 above)
Comment (paramat) [reply mods/default/nodes.lua]: I agree, will do.

Comment (paramat) [reply mods/default/nodes.lua]: Done.

**Comment by paramat** [comment]
Updated as requested and retested.

**Comment by paramat** [comment]
3f14d1012dfc095b5019c805fd240ae5aaad4200

**Pull request closed without merging**